packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -241,13 +263,48 @@ export const Table: React.FunctionComponent = () => {
             icon: 'trash',
             type: 'icon',
             color: 'danger',
-            onClick: console.log
+
+            onClick: item => {
+              setIsModalVisible(true)
+              setDeleteId([item.id])
+            }
           }
         ]
       }
     ]
   }, [resource])
 
+  const ConfirmDeleteModal = () => {
+    const closeModal = () => setIsModalVisible(false)
+
+    if (isModalVisible) {
+      return (
+        <EuiConfirmModal
+          title={`Do you want to delete the selected item${
+            selectedItems.length > 1 ? 's' : ''
+          }`}
+          onCancel={closeModal}
+          onConfirm={() => {
+            deleteItem()
+            closeModal()
+          }}
+          cancelButtonText="No, don't do it"
+          confirmButtonText="Yes, do it"
```

Comment:
The confirm button text should be `Delete products` or `Delete categories` or `Delete collections`. Also should be singular if just one is being deleted. You can check the resource to get the singular or plural names

Reply:
the `resource.name` doesn't have plural, can i use `resource.slugPlural`? or use one of the camelcase ones and use `toLowerCase()`